src/components/new-safe/create/logic/index.ts

```diff
@@ -112,9 +107,10 @@ export const encodeSafeCreationTx = async ({
   chain,
 }: SafeCreationProps & { chain: ChainInfo }) => {
   const readOnlySafeContract = await getReadOnlyGnosisSafeContract(chain, LATEST_SAFE_VERSION)
-  const readOnlyProxyContract = await getReadOnlyProxyFactoryContract(chain.chainId, LATEST_SAFE_VERSION)
-  const readOnlyFallbackHandlerContract = await getReadOnlyFallbackHandlerContract(chain.chainId, LATEST_SAFE_VERSION)
+  const readOnlyProxyContract = await getReadOnlyProxyFactoryContract(LATEST_SAFE_VERSION)
+  const readOnlyFallbackHandlerContract = await getReadOnlyFallbackHandlerContract(LATEST_SAFE_VERSION)
 
+  // @ts-ignore union type is too complex
```

Comment:
I counted 10 ts-ignores added in this PR, are we planning to fix them?

Reply:
This has to be fixed in the protocol-kit. ([Discussion on Slack](https://5afe.slack.com/archives/C0660KAEG49/p1718626730277609))